src/XRP/model/xrp-payment.ts

```diff
@@ -1,22 +1,33 @@
+import { Utils } from 'xpring-common-js'
 import { Payment } from '../Generated/web/org/xrpl/rpc/v1/transaction_pb'
 import XRPCurrencyAmount from './xrp-currency-amount'
 import XRPPath from './xrp-path'
+import XRPLNetwork from '../../Common/xrpl-network'
 
 /*
  * Represents a payment on the XRP Ledger.
  * @see: https://xrpl.org/payment.html
  */
-// TODO(amiecorso): Modify this object to use X-Address format.
 export default class XRPPayment {
+  /**
+   * The address and (optional) destination tag of the account receiving the payment, encoded in X-address format.
+   * See https://xrpaddress.info/.
+   */
+  destinationXAddress: string | undefined
+
   /**
    * Constructs an XRPPayment from a Payment.
    *
    * @param payment a Payment (protobuf object) whose field values will be used
    *                to construct an XRPPayment
+   * @param xrplNetwork The XRPL network from which this object was retrieved, defaults to XRPLNetwork.Main (Mainnet).
```

Comment:
Is there a reason to default here? Why not force users to explicitly say where this came from?

Reply:
That's a good point... `xrplNetwork` defaults in the client classes, which makes sense, but since it's the client classes that feed their value to these objects, we may as well not default.  One reason to default is to avoid having to update all of the existing constructor calls, but that's not a good reason.

Reply:
It'll also make our Java code cleaner, which doesn't deal well with default arguments.  I'll update this.